Addition FOR  Update to sample list view PR #986

jbflo opened the pull request:
Also adding  Small  Fix  on Observer Dialog Form

marcus-oscarsson commented:
@jbflo looks like a few too many commits :), we can have a look together tomorrow or next week if you like.

Commit:
Fix for Sample being Slow

small fix on SelectionRubberBand


Small FIX for when sc Contents Differ Sample List


Put Observer Input Field inside Form Submit component

jbflo commented:
> @jbflo looks like a few too many commits :), we can have a look together tomorrow or next week if you like.

@marcus-oscarsson I cheery Pick the commit i wanted to add instead of rebase.

marcus-oscarsson commented:
Thanks, its a bit shorter now :)
